python_ta/contracts/__main__.py

```diff
@@ -0,0 +1,109 @@
+import os.path
+import sys
+import traceback
+import types
+from typing import List, TextIO, Tuple
+
+import click
+
+from . import check_all_contracts
+
+
+@click.command()
+@click.argument("file", type=click.File(mode="r"))
+@click.argument("extra-mod-names", nargs=-1)
```

Comment:
Yeah, you can make this an option. You can make it a comma-separated list of module names.